src/components/icons/chevron-right.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const ChevronRight: React.FC = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={1.5}
```

Comment:
small deatil but I would set the stroke-width to 2.5 for the chevron right and down

Reply:
fixed